Makefile

```diff
@@ -119,8 +119,10 @@ build.version:
 	@echo "$(VERSION)" > $(OUTPUT_DIR)/version
 
 # Change how CRDs are generated for CSVs
+ifneq ($(REAL_HOST_PLATFORM),darwin_arm64)
```

Comment:
@subhamkrai Do we even need these vars on other platforms?

Reply:
I don't think we need in any other platforms, for Linux distributions I don't think.

Reply:
per separate conversion, will leave it like this, and we can remove later when csv generation is moved out